src/partials/launch-subscribe.hbs

```diff
@@ -0,0 +1,74 @@
+{{#with subscribe}}
+<section class="subscribe-container">
+    <header>
+      <button type="button" class="subscribe-container__close-button">
+        <img src="../../../icons/cross.svg" height="10" width="10" alt=""
+           class="subscribe-container__close-icon">
+      </button>
+      <div class="subscribe-container__header-container">
+        <p class="subscribe-container__header-message">
+          {{this.subtitle}}
+        </p>
+        {{#if this.checklists}}
+          <img src="../../../icons/invalid-name.svg" height="16" width="34" alt=""
+             class="subscribe-container__header-icon">
+        {{/if}}
+        {{#unless this.checklists}}
+          <img src="../../../icons/confused-face.svg" height="12" width="47" alt=""
+               class="subscribe-container__header-icon">
+        {{/unless}}
+      </div>
+      <h2 class="subscribe-container__heading">
+        {{this.title}}
+      </h2>
+    </header>
+    <main>
+      {{#if this.checklists}}
+        <div class="subscribe-container__checklists-container checklists-container">
```

Comment:
this is a mix (https://ru.bem.info/methodology/quick-start/#%D0%BC%D0%B8%D0%BA%D1%81)
for mix, the block at first, than element
so, `checklists-container subscribe-container__checklists-container`